code/src/store/slices/localCart.ts

```diff
@@ -0,0 +1,76 @@
+import { PayloadAction, createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+
+import { initialCart } from "@/constants/cart";
+import { useAppSelector } from "@/hooks/use-redux/useRedux";
+import cartApi from "@/store/api/cartApi";
+import { sliceNames } from "@/store/constants";
+import { RootState } from "@/store/store";
+import { CartItem } from "@/types/cart.types";
+import getLocalStorageCart from "@/utils/get-local-storage-cart/getLocalStorageCart";
+
+const initialState = getLocalStorageCart();
+
+export const removeFromLocalCart = createAsyncThunk(
+  "localCart/removeFromCart",
+  async (cartItem: CartItem, storeApi) => {
+    const { localCart } = storeApi.getState() as RootState;
+
+    const existingItem = localCart.items.find(
+      (item) => item.productId === cartItem.productId
+    );
+
+    if (existingItem) {
+      storeApi.dispatch(localCartSlice.actions.removeFromLocalCart(cartItem));
+    } else {
+      throw new Error("Item not found in cart");
+    }
+  }
+);
+
+export const addToLocalCart = createAsyncThunk(
+  "localCart/removeFromCart",
```

Comment:
addToLocalCart uses localCart/`removeFromCart` is it okay?

Reply:
Thank you, I didn't notice it